catalog/resource_grant_test.go

```diff
@@ -834,3 +857,23 @@ func TestResourceGrantModelGrantCreate(t *testing.T) {
 		`,
 	}.ApplyNoError(t)
 }
+
+func TestResourceGrantUpdateSpacesNoUpdate(t *testing.T) {
+	qa.ResourceFixture{
+		Resource: ResourceGrant(),
+		ID:       "table/foo.bar.baz/me",
+		InstanceState: map[string]string{
+			"table":                 "foo.bar.baz",
+			"principal":             "me",
+			"privileges.#":          "1",
+			"privileges.3182106578": "ALL_PRIVILEGES", // this indexer seems like a huge hack
```

Comment:
I want to also call out this indexer logic that doesn't make sense to me, but this seemingly "magic" value didn't change between test runs; I would love to know how to configure this in a more stable fashion